src/NServiceBus.NHibernate.AcceptanceTests/NServiceBus.NHibernate.AcceptanceTests.csproj

```diff
@@ -25,8 +25,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="\**\When_handling_concurrent_messages.cs" />
-    <Compile Remove="\**\NSB.AcceptanceTests\DelayedDelivery\When_deferring_a_message.cs" />
+    <Compile Remove="$(PkgNServiceBus_AcceptanceTests_Sources)/**/When_handling_concurrent_messages.cs" />
+    <Compile Remove="$(PkgNServiceBus_AcceptanceTests_Sources)/**/When_deferring_a_message.cs" />
```

Comment:
A comment would be good as why they are excluded

Reply:
I actually have no idea ;)

Reply:
Then, shouldn't we figure it out?

Reply:
I call `out-of-scope` on it.